roles/ipaserver/tasks/uninstall.yml

```diff
@@ -55,7 +55,7 @@
   failed_when: uninstall.rc != 0 and uninstall.rc != 1
   changed_when: uninstall.rc == 0
 
-#- name: Remove IPA server packages
-#  ansible.builtin.package:
-#    name: "{{ ipaserver_packages }}"
-#    state: absent
+# - name: Remove IPA server packages
+#   ansible.builtin.package:
+#     name: "{{ ipaserver_packages }}"
+#     state: absent
```

Comment:
Please remove these lines. A role my not remove packages.

Reply:
Fixed.